typednavigation/build.gradle

```diff
@@ -66,12 +66,15 @@ android {
 
 dependencies {
 
-    implementation 'androidx.core:core-ktx:1.6.0'
-    implementation 'androidx.appcompat:appcompat:1.3.1'
-    implementation 'com.google.android.material:material:1.4.0'
-    implementation 'androidx.navigation:navigation-compose:2.4.0-alpha10'
-    implementation 'androidx.compose.compiler:compiler:1.0.3'
+    implementation 'androidx.core:core-ktx:1.7.0'
+    implementation 'androidx.appcompat:appcompat:1.4.1'
+    implementation 'com.google.android.material:material:1.5.0'
+    implementation 'androidx.navigation:navigation-ui-ktx:2.3.5'
+    implementation 'androidx.navigation:navigation-compose:2.4.0-rc01'
```

Comment:
do you need it? because a library shouldn't use unstable versions unless you need it